style.css

```diff
@@ -572,4 +672,104 @@ fieldset textarea {
     background-repeat: no-repeat;
     background-size: cover;
   }
+
+  /* Modal popup Window */
+  .divClass {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: rgba(255, 255, 255, 0.16);
+    display: flex;
+    flex-direction: column;
+    width: 1156px;
+    height: 1176px;
+  }
```

Comment:
- [ ] Please refer to the given design from the earlier review and fix the styling spacing issues. Currently, they're too large on this screen size!
- [ ] Also, please fix the images

>Take a look at the screenshot down below!
![Screenshot](https://user-images.githubusercontent.com/47625407/123311264-413eaf00-d527-11eb-867a-d4c63dd5daf7.png)

Reply:
![Screen Shot 2021-06-24 at 11 31 08](https://user-images.githubusercontent.com/67211919/123317717-f22b5680-d4f3-11eb-8511-9bec4dd3d175.png)

@MahmoudBakr23 Thanks can you help me? I set the dimensions like in the Figma template said.

Reply:
You can change the media query for that to fit perfectly 👌🏻